client/src/App/index.js

```diff
@@ -1,34 +1,45 @@
-import React, { useState } from 'react';
+import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-import { Layout } from '../components';
+import { Layout, Header } from '../components';
 import { Current, Home, Login, PastBills, Profile } from '../pages';
+import useAuth from '../utils/useAuth';
+import AuthContext from '../context';
```

Comment:
using a context here is an overkill, just pass down the methods as pops to the login page!